src/app/controllers/borrows.controller.js

```diff
@@ -91,28 +92,45 @@ module.exports = {
 
     insert: async(req, res, next) => {
         try {
-            const getBorrowUser = await borrowingHistory.findAll({
+            const nim = req.body.nim
+            const TODAY_START = moment().format('YYYY-MM-DD 00:00');
+            const TODAY_END = moment().format('YYYY-MM-DD 23:59');
```

Comment:
make sure timezone in +7 and change it to UTC
moment.tz('YYYY-MM-DD 00:00:00', 'YYYY-MM-DD HH:mm:ss', 'jakarta+7').utc().format()